core/src/software/aws/toolkits/core/credentials/ToolkitCredentialsProvider.kt

```diff
@@ -120,10 +120,11 @@ class ToolkitBearerTokenProvider(val delegate: ToolkitBearerTokenProviderDelegat
         fun ssoIdentifier(startUrl: String, region: String = DEFAULT_SSO_REGION) = "sso;$region;$startUrl"
 
         // TODO: For AWS Builder ID, we only have startUrl for now instead of each users' metadata data i.e. Email address
-        fun ssoDisplayName(startUrl: String) = if (startUrl == SONO_URL) {
+        fun ssoDisplayName(startUrl: String, region: String) = if (startUrl == SONO_URL) {
             message("aws_builder_id.service_name")
         } else {
-            message("iam_identity_center.service_name", extractOrgID(startUrl))
+            val ssoSessionProfileName = "${extractOrgID(startUrl)}-$region"
```

Comment:
as discussed offline, naming will be addressed separately

Reply:
Sure, already created a ticket for it